README.md

```diff
@@ -197,6 +197,7 @@ module "trino" {
   source = "github.com/fredrikhgrelland/terraform-nomad-trino.git?ref=0.3.0"
 
   depends_on = [
+    module.postgres,
```

Comment:
Might be good to mention that for simplicity, the same `postgres` instance uses for both `hive` and `trino` (my initial thoughts)

Reply:
Will add!

Reply:
Done